docs/Ada_Drivers_Library-vs-bb_runtimes.md

```diff
@@ -0,0 +1,32 @@
+We understand that there might be some confusion between the two repositories
+`Ada_Drivers_Library` and `bb-runtimes`. Who does what? What should go where?
+This page will try to provide and answer.
+
+Don't hesitate to ask more questions on this subject or suggest improvements to
+this page.
+
+# Ada_Drivers_Library vs bb-runtimes
+
+The run-times (`bb-runtimes` repository) provide an implementation of Ada
+language features as defined by the standard. These features may include
+interrupt handling, tasking and time services that are usually provided by what
+is called an RTOS in the C/C++ world.
+
+The run-times will need to control a few peripherals to provide those features
+(interrupt controller, one timer, some CPU registers), but it won't touch the
+other peripherals (I2C, SPI, GPIOs, Ethernet, etc.).
+
+`Ada_Drivers_Library` on the other side is a library for firmware development
+in Ada. It provides drivers for remaining peripherals on the system as well
+drivers for external components and also middleware services. Note that
+`Ada_Drivers_Library` may rely on Ada run-time features but it does not meant
+that the run-time has to be from the `bb-runtimes` repository, there are other
+implementation possible depending on the architecture.
+
```

Comment:
OK I see: `The limit between the two projects`, that must be Franglish. Maybe `The border` or `The boundary`?